src/main/java/io/confluent/connect/jdbc/sink/BufferedRecords.java

```diff
@@ -177,6 +178,7 @@ public BufferedRecords(
   }
 
   public void close() throws SQLException {
+    log.info("Closing BufferedRecords with preparedStatement: {}", preparedStatement);
```

Comment:
Just found this PR when reporting this issue https://github.com/confluentinc/kafka-connect-jdbc/issues/746

Consider changing `info` level to `debug` or `trace` in this case.